CONTRIBUTING.md

```diff
@@ -95,6 +95,22 @@ cloud resources. Replay files scrub sensitive information.
 [Send your PR](#making-a-pull-request) without the replay files, and we can
 generate new ones to be used by others.
 
+### Dependencies
+
+The Go CDK has a policy to only depend on code that is under a license that is
+categorized under [`notice`][notice license],
+[`permissive`][permissive license], or [`unencumbered`][unencumbered license]
+in the [Google Open Source Licenses][] documentation. This is enforced with a
+Travis build check that verifies that every module depended on is in the
+[`alldeps` file][]. Do not add new lines to the file unless you have verified
```

Comment:
"Do not add new lines..." is a little confusing since we don't directly add lines to this file. Maybe something like "Do not add new (direct or indirect) dependencies unless ..."?

Reply:
Done.